third_party/move/move-compiler-v2/tools/testdiff/src/main.rs

```diff
@@ -53,10 +56,7 @@ const V2_E2E_ROOT: &str = "move-compiler-v2/transactional-tests/tests";
 /// Ignored directories in the move_check tree.
 static IGNORED_UNIT_PATHS: Lazy<BTreeSet<String>> = Lazy::new(|| {
     let mut set = BTreeSet::new();
-    set.insert("parser".to_string());
```

Comment:
Presumably, you needed to not ignore this because there were some parser tests you wanted to match against for the attributes stuff?

This change means that we would either (a) need to port over unmatched parser tests into v2, even though we currently use the exact same parser, (b) be okay with unmatched parser tests.

Reply:
At some point we need to worry about whether we are invoking the parser correctly in compiler v2.  For this reason, it's not a good idea to ignore the parser tests forever.

For example, are we recognizing V2 language extensions?  Test annotations?  etc.

Reply:
Sure, but the same logic applies to expansion as well, which we currently ignore ...

It would be helpful for the reviewer when there is such a large change, the PR description mentions why it is being done (parser tests need to unignored here because ..., we will port parser tests as follow up, etc). I agree that "at some point we need to worry ...", my question was just to understand why we are doing it here.

Reply:
Yes, here we are removing it because the attribute tests are under parser.